### PR TITLE
Language name with arbitrary casing

### LINKED ISSUE #243: Allow languages to use arbitrary casing in their names
I tried changing _gobstones_ to _Gobstones_ but nothing happened.

### PR DESCRIPTION
**Pull request by faloi**
Closes #243

### CODE REVIEW

**Comment by flbulgarelli** [comment]
This change will break the import process, for instance. If you want to have this feature, please ensure that everytime language find by name, search is case insensitive.

**Comment by faloi** [comment]
Guides and exercises search is already case insensitive. Do we have another kind of search?

**Comment by flbulgarelli** [comment]
> Guides and exercises search is already case insensitive. Do we have another kind of search?

Yes, as part of import process.

**Comment by faloi** [comment]
> This change will break the import process, for instance.

We should have a test for that :stuck_out_tongue_winking_eye:

File: app/models/import.rb [hunk 1]
```diff
@@ -40,7 +40,7 @@ def read_extra!(dir)
   def read_meta!(dir)
     meta = read_yaml_file(File.join(dir, 'meta.yml'))
 
-    guide.language = Language.find_by!(name: meta['language'].downcase)
+    guide.language = Language.find_by_name! meta['language']
```
Comment (flbulgarelli) [inline app/models/import.rb]: That syntax is deprecated. Please use `find_by(name:...)`

**Pull request merged**